### PR TITLE
fix: drop python 3.6 in python-multi

### PR DESCRIPTION
**Pull request by parthea**

### CODE REVIEW

**Review by engelke — APPROVED** [review]

File: python/googleapis/python-multi/Dockerfile [hunk 1]
```diff
@@ -146,12 +144,12 @@ RUN for PYTHON_VERSION in 2.7.18 3.6.15 3.7.12 3.8.12 3.9.7 3.10.0; do \
   && rm -r "$GNUPGHOME"
 
 
-# Install pip on Python 3.6 only.
+# Install pip on Python 3.10 only.
```
Comment (dandhlee) [inline python/googleapis/python-multi/Dockerfile:147]: Can we use Python3.8 or 3.9 instead? 3.10 is not thoroughly tested or supported.

Comment (parthea) [reply python/googleapis/python-multi/Dockerfile:147]: Addressed in [8f71a30](https://github.com/googleapis/testing-infra-docker/pull/231/commits/8f71a30710254a529923d2503dd4c28f1af58bec)

File: python/googleapis/python-multi/Dockerfile [hunk 2]
```diff
@@ -177,7 +185,7 @@ RUN pip install --no-cache-dir virtualenv
 # Setup Cloud SDK
 ENV CLOUD_SDK_VERSION 360.0.0
 # Use system python for cloud sdk.
-ENV CLOUDSDK_PYTHON python3.6
+ENV CLOUDSDK_PYTHON python3.7
```
Comment (dandhlee) [inline python/googleapis/python-multi/Dockerfile:188]: Could this be bumped to 3.8?

**Review by dandhlee — APPROVED** [review]
LGTM, one minor nit.

File: python/googleapis/python-multi/Dockerfile (hunk 2 above)
Comment (parthea) [reply python/googleapis/python-multi/Dockerfile]: Done

**Pull request merged**